tests/integration/test_deposit_switch.py

```diff
@@ -0,0 +1,57 @@
+from tests.integration.util import (
+    create_client,
+)
+
+access_token = None
+target_account_id = None
+deposit_switch_id = None
+
+
+def setup_module(module):
```

Comment:
In a way, the setup_module here is testing all of the functions besides `get`. I would propose limiting setup_module to creating an item (perhaps with `client.Sandbox.public_token.create(SANDBOX_INSTITUTION, ['auth', 'identity'])`) as a way of getting the access token. Then, I would suggest doing a `test_full_flow` that tests `create`, `token/create`, and `get` in sequence. This feels more concise and avoids being redundant. WDYT?